packages/ffe-form-react/src/PhoneNumberField.md

```diff
@@ -0,0 +1,9 @@
+Telefonnummer med landskode
+
+```js
+<PhoneNumberField 
+    phone={"123123123"} 
+    countryCode={"47"}
```

Comment:
More a style preference, but for Strings you can skip the curly braces:

```jsx
countryCode="47"
phone="12345678"
```

Reply:
This should be corrected by Prettier, if i remember correctly!